### ISSUE TITLE
Suggestions for improving the guide

### ISSUE THREAD

**Issue by falexwolf** [laminlabs/nextflow-lamin#29]
Shall we remove this? It's now running in a script.

<img width="834" alt="image" src="https://github.com/laminlabs/nextflow-lamin-usecases/assets/16916678/58f8242b-b51f-483e-9c87-7d6c6ce93eb5">

**Comment by falexwolf** [comment]
I'm questioning the value of the "pre-populated LaminDB instance".

* Pro: It makes the lineage graph more interesting.
* Con: It requires to look into a notebook and it all appears quite complicated. It's also weird that the input path for the actual nextflow command is NOT derived from a folder-like artifact, but hardcoded as a string.

<img width="645" alt="image" src="https://github.com/laminlabs/nextflow-lamin-usecases/assets/16916678/ba87f791-e422-47dd-ba86-d49966a9ab74">

(Also: Let's remove the "Setup" header so that it looks less complicated. This is what we aligned on a while back I think.)

**Comment by falexwolf** [comment]
Consistently hide cell outputs everywhere:
<img width="824" alt="image" src="https://github.com/laminlabs/nextflow-lamin-usecases/assets/16916678/d09fa7f3-80fd-42c5-824e-f2538b37ffd3">

We're meanwhile doing this in most guides because it makes things look much less noisy.

**Comment by falexwolf** [comment]
Remove the random comment about a bulk-rnaseq pipeline:
<img width="832" alt="image" src="https://github.com/laminlabs/nextflow-lamin-usecases/assets/16916678/2d57e7a1-605e-4c73-b8e7-00f4cbf80ecc">

**Comment by falexwolf** [comment]
Yes, I think we agreed on removing the "Setup" header: https://laminlabs.slack.com/archives/C046F63LJJ2/p1705483044994189

**Comment by Zethson** [comment]
I think all of this is kinda outdated.
